CONTRIBUTING.md

```diff
@@ -14,3 +14,4 @@ license the work to the project under the project's open source license. Whether
 state this explicitly, by submitting any copyrighted material via pull request, email, or 
 other means you agree to license the material under the project's open source license and 
 warrant that you have the legal authority to do so.
+Thank you!
```

Comment:
Again, i don't think this particularly improve the contribution guide and the example. I wouldn't add this.

Reply:
+1